### DISCUSSION TITLE
Reverse Alias with sl suffix regex auto create reverse alias,

### DISCUSSION THREAD

**Discussion by Wolf1098** [simple-login/app#1892] [Feature Requests]
if you could find a way to guide us to create something@somedomain.com > something+ (can assume that first non escaped plus is @ and following is zeros maybe?) somedomain+com@out.simplylogin.com

one way to denote the desired contact, from the desired alias, to be sending email from. is find something alternative to :, and have one long part before @ made up of two sections, that breaks into 2 email address, the contact, and the outgoing alias, or put it as a subdomain in front of out.somethingsimplelogin.com like support+simplelogin+com@wolf1098+wolfietech.xyz.out.simplelogin.com

**Comment by Wolf1098** [comment]
There could be a script made that based on that way the final regex worked, constructed valid outgoing emails, based on your contacts original email, and that could be done transparently in protonmail

**Reply by Wolf1098** [reply]
all that would remain after is put the sending like foo@bar.com email through the normal authorized reverse alias senders lists, if foo is authorized for wolf1098+wolfietech.xyz.out.simplelogin.com, allow transformation of support+simplelogin+com@ to support@simplelogin.com as the to

**Comment by crazy-matt** [comment]
might be a duplicate of https://github.com/simple-login/app/discussions/1892
